arm-network/2017-06-01/swagger/loadBalancer.json

```diff
@@ -396,10 +396,11 @@
         },
         "protocol": {
           "type": "string",
-          "description": "The transport protocol for the external endpoint. Possible values are 'Udp' or 'Tcp'",
+          "description": "The transport protocol for the external endpoint. Possible values are 'Udp,' 'Tcp,' or 'All.'",
           "enum": [
             "Udp",
-            "Tcp"
+            "Tcp",
```

Comment:
I added a comma to the end of the line. That's why it shows up like that. The comma is required in order to add another value.